### DISCUSSION TITLE
[@types/paho-mqtt] Missing reconnect flag on ConnectionOptions interface

### DISCUSSION THREAD

**Discussion by touchofevil-dev** [DefinitelyTyped/DefinitelyTyped#57527] [Issues with a @types package] [answered]
[ConnectionOptions interface](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/paho-mqtt/index.d.ts#L118) for [paho-mqtt library](https://github.com/eclipse/paho.mqtt.javascript/blob/master/src/paho-mqtt.js#L1945) seems to be missing a boolean property "reconnect".

This causes the property to not show up and also code to not compile when using the library with TS.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
PR is ready #57527 but missing a URL to the documentation in your description.
